packages/flutter_tools/templates/app/linux.tmpl/my_application.cc.tmpl

```diff
@@ -23,6 +23,8 @@ static void my_application_activate(GApplication* application) {
   gtk_widget_show(GTK_WIDGET(window));
 
   g_autoptr(FlDartProject) project = fl_dart_project_new();
+  MyApplication* my_app = MY_APPLICATION(application);
```

Comment:
Rename `my_app` to `self` and put at the top of the function (see other functions for convention).